EU4toV2/Source/Configuration.cpp

```diff
@@ -41,6 +41,10 @@ void Configuration::instantiate(std::istream& theStream, bool (*DoesFolderExist)
 		verifyVic2DocumentsPath(Vic2DocumentsPath, DoesFolderExist);
 		LOG(LogLevel::Info) << "Vic2 documents path: " << Vic2DocumentsPath;
 	});
+	registerKeyword("targetGameModPath", [this](const std::string& unused, std::istream& theStream) {
```

Comment:
"hybridModSource" - we're creating a hybrid mod using this one as source.

Reply:
This was recycled from another converter. It points to where Victoria II mods are stored.

Reply:
Though, I'm 99% certain that it was only needed with the "any mod" mind frame. So I'm removing it entirely.

Reply:
Nope, I was wrong. I need it to tell the Converter that HPM folder is inside the `C:\Vic2ModPath`, unless we can assume that the path is always at `C:\Vic2Path\mod` which is a bit risky. As someone once said people do all kinds of crazy things.